Lib/netrc.py

```diff
@@ -33,30 +35,29 @@ def _read_char(self):
         return ch
 
     def get_token(self):
+        self.dontskip = False
         if self.pushback:
             return self.pushback.pop(0)
         token = ""
-        fiter = iter(self._read_char, "")
-        for ch in fiter:
-            if ch in self.whitespace:
+        enquoted = False
+        while ch := self._read_char():
```

Comment:
Was this refactoring necessary to fix the bug? It's rather difficult to read the diff with so many lines reshuffled. If I were to guess, this might be the reason people are postponing doing reviews on this PR.